parts/windows/windowscsehelper.ps1

```diff
@@ -335,4 +337,37 @@ function Install-Containerd-Based-On-Kubernetes-Version {
     $ContainerdUrl = $ContainerdUrl + $containerdPackage
   }
   Install-Containerd -ContainerdUrl $ContainerdUrl -CNIBinDir $CNIBinDir -CNIConfDir $CNIConfDir -KubeDir $KubeDir
+}
+
+function Logs-To-Events {
+    $count = $args.Count
+    $cmd = [string]$args[1..$count]
+    $cmd = $cmd.Replace('\', '\\')
+
+    $task = $args[0]
+    $eventsFileName=[DateTimeOffset]::UtcNow.ToUnixTimeMilliseconds()
+
+    $startTime=$(Get-Date -Format "yyyy-mm-dd HH:mm:ss.fff")
+    Invoke-Expression $cmd
+    $ret=$?
+    $endTime=$(Get-Date -Format "yyyy-mm-dd HH:mm:ss.fff")
+    
+    $json_string = @"
+    {
+        "Timestamp": "$startTime",
+        "OperationId": "$endTime",
+        "Version": "1.23",
+        "TaskName": "$task",
+        "EventLevel": "Informational",

```

Comment:
Could we change the EventLevel to error for failures?

Reply:
Sure. Change the EventLevel to error for failures.